### ISSUE TITLE
ValueError: only one element tensors can be converted to Python scalars

### ISSUE THREAD

**Issue by wg1996** [pytorch/pytorch#22674]
When I try transfer a mobilenet_v2 model from pytorch to onnx, ValueRrror has appeared.
ValueError: only one element tensors can be converted to Python scalars
 How can I sovled it? Here is my code:
```
import torch
import torchvision
import torch.onnx
from torch.autograd import  Variable

# An instance of your model
model = torchvision.models.mobilenet_v2(pretrained=True)
print(model)

x = torch.randn(1, 3, 224, 224,requires_grad=True)
# Export the model
torch_out = torch.onnx._export(model, x, "mobilenet_v2.onnx", export_params=True)
```

**Comment by ezyang** [comment]
Confirmed I am able to reproduce on master.

**Comment by ezyang** [comment]
The backtrace is:

```
Traceback (most recent call last):
  File "moo.py", line 12, in <module>
    torch_out = torch.onnx._export(model, x, "mobilenet_v2.onnx", export_params=True)
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/__init__.py", line 26, in _export
    result = utils._export(*args, **kwargs)
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/utils.py", line 405, in _export
    _retain_param_name, do_constant_folding)
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/utils.py", line 316, in _model_to_graph
    _disable_torch_constant_prop=_disable_torch_constant_prop)
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/utils.py", line 225, in _optimize_graph
    graph = torch._C._jit_pass_onnx(graph, operator_export_type)
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/__init__.py", line 57, in _run_symbolic_function
    return utils._run_symbolic_function(*args, **kwargs)
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/utils.py", line 640, in _run_symbolic_function
    return op_fn(g, *inputs, **attrs)
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/symbolic_opset9.py", line 211, in symbolic
    dim, keepdim = sym_help._get_const(dim, 'i', 'dim'), sym_help._get_const(keepdim, 'i', 'keepdim')
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/symbolic_helper.py", line 114, in _get_const
    return _parse_arg(value, desc)
  File "/data/users/ezyang/pytorch-tmp/torch/onnx/symbolic_helper.py", line 73, in _parse_arg
    return int(tval)
ValueError: only one element tensors can be converted to Python scalars
```

**Comment by ezyang** [comment]
@lara-hdr could you please take a look? This codepath was introduced in #19294

**Comment by lara-hdr** [comment]
@ezyang in #19294 the helper functions were moved from symbolic.py to symbolic_helper.py, but the code was unchanged for the functions in question (mean , _get_const and _parse_arg).

**Comment by spandantiwari** [comment]
@wg1996 - thanks for reporting this. This issue has been fixed in PyTorch master already, and I am able to export `torchvision.models.mobilenet_v2` to ONNX.  Could you please try the latest PyTorch master or install the nightly build to verify this.
Feel free to reopen if you still see this issue. Thanks.

**Comment by wg1996** [comment]
@spandantiwari  @lara-hdr Thank you very much for your kind answer, I will try the latest version when I am free.

**Comment by GiscardBiamby** [comment]
Had this problem with pytorch 1.1.0, fixed after upgrading to 1.2

**Comment by zubairahmed-ai** [comment]
> Had this problem with pytorch 1.1.0, fixed after upgrading to 1.2

Facing the same issue my torch is `torch==1.1.0` and pip says requirement already satisfied there is no `1.2` version?
